internal/build/docker_test.go

```diff
@@ -167,10 +165,10 @@ func (f *testFixture) assertFileInImageWithContents(tag digest.Digest, path stri
 	io.Copy(output, out)
 
 	if strings.Contains(output.String(), "No such file") {
-		f.t.Errorf("Expected to find file %s in container. Got: %s", path, output)
+		f.t.Errorf("Expected to find file in container. Got: %s", output)
```

Comment:
think this conditional is useless now: this is the bad output from `cat`, and `grep` will eat the output of `cat`, right?